blocks/bricks/base.py

```diff
@@ -210,12 +210,14 @@ def __getattr__(self, name):
         if 'properties' in self.__dict__ and name in self.properties:
             return property(create_unbound_method(
                 getattr(self, self.properties[name]), self.brick))
-        raise AttributeError
+        raise AttributeError("Cannot find '%s' property in brick %s" %
+                             (name, str(self)))
 
     def __setattr__(self, name, value):
         # Mimic behavior of read-only properties
         if 'properties' in self.__dict__ and name in self.properties:
-            raise AttributeError("can't set attribute")
+            raise AttributeError("Can't set attribute '%s' in brick %s" %
```

Comment:
Yeah, but Python error message have to be very generic because no detailed information is available at the programming language level. We in Blocks could be more generous.

But okay, I also can live with this message, it's better than simply `AttributeError`.